syncd/RedisClient.cpp

```diff
@@ -893,17 +893,46 @@ void RedisClient::processFlushEvent(
             SWSS_LOG_THROW("unknown fdb flush entry type: %d", type);
     }
 
-    for (int flush_static: vals)
+    // If has a lot of macs(example:217600) and use lua scripts, will cause REDIS BUSY.
+    // Change to this without atomicity operation.
+    const auto &keys = m_dbAsic->keys(pattern);
+    size_t countFlushed = 0;
+    if (vals.size() == 2 && portStr.empty())
     {
-        swss::RedisCommand command;
-
-        command.format(
-                "EVALSHA %s 3 %s %s %s",
-                m_fdbFlushSha.c_str(),
-                pattern.c_str(),
-                portStr.c_str(),
-                std::to_string(flush_static).c_str());
-
-        swss::RedisReply r(m_dbAsic.get(), command);
```

Comment:
I am trying to understand the root reason of "REDIS BUSY". If m_dbAsic is using redis pipeline, you could explicitly call its flush function to improve the redis responsibilities.